### PR TITLE
Add upstream problem list to some hotspot testcases

### LINKED ISSUE #2712: Some testcase in default openjdk problemlist was not excluded by default in aqa-tests
There are tests which are in the default exclude list which are not being exclude by default in aqa-tests. For example:

[compiler/rtm/locking/TestRTMAbortRatio.java](https://github.com/adoptium/jdk8u/blob/master/hotspot/test/ProblemList.txt)

The reason is problemlist file hasn't be added to some testcases in hotspot component.

### PR DESCRIPTION
**Pull request by sophia-guo**
Close  #2712 

Signed-off-by: Sophia Guo <sophia.gwf@gmail.com>

### CODE REVIEW

**Comment by sophia-guo** [comment]
Grinder with jdk_jre: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/1029/testReport/

**Review by smlambert — APPROVED** [review]
LGTM

File: openjdk/playlist.xml [hunk 1]
```diff
@@ -48,6 +48,8 @@
 	-w $(Q)$(REPORTDIR)$(D)work$(Q) \
 	-r $(Q)$(TEST_RESROOT)$(D)report$(Q) \
 	-jdk:$(Q)$(TEST_JDK_HOME)$(Q) \
+	-exclude:$(Q)$(JTREG_HOTSPOT_TEST_DIR)$(D)ProblemList.txt$(Q) \
+    -exclude:$(Q)$(TEST_RESROOT)$(D)$(PROBLEM_LIST_FILE)$(Q) \
```
Comment (llxia) [inline openjdk/playlist.xml:52]: Could you please fix the indentation? The same for the rest of PR. Thanks

Comment (sophia-guo) [reply openjdk/playlist.xml]: Done.

Comment (llxia) [reply openjdk/playlist.xml]: Thanks

**Review by llxia — APPROVED** [review]
LGTM

**Pull request merged**